go.mod

```diff
@@ -44,9 +44,12 @@ require (
 	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20231202142526-55ffb0092afd
 	github.com/google/uuid v1.6.0
 	github.com/minio/minio-go/v7 v7.0.66
+	github.com/notaryproject/notation-core-go v1.0.2
+	github.com/notaryproject/notation-go v1.1.0
 	github.com/onsi/gomega v1.31.1
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/go-digest/blake3 v0.0.0-20231025023718-d50d2fec9c98
+	github.com/opencontainers/image-spec v1.1.0-rc5
```

Comment:
this can be update to v1.1.0

Reply:
Resolved in [198f57b](https://github.com/fluxcd/source-controller/pull/1075/commits/198f57bb272fc8779b964bab15e53760b0025f66).